### ISSUE TITLE
Onboard a new team member

### ISSUE THREAD

**Issue by damianrr** [aws-amplify/amplify-cli#3504]
**Which Category is your question related to?**
Team

**Amplify CLI Version** 
4.13

Hello all,

Couple of questions if you're so kind.
I've an amplify app deployed on my Amplify Console, yet recently I've noticed that instead of one app it shows as 3 instead of one (the main one and one per amplify env?) see first image attached. I could swear that this is a recent thing, I've always had only one app in my Console, so, my first question would be: Did I do something wrong or this is normal?
On the other hand, I've a new teammate, I've always worked alone on this project, I've been reading this: https://aws-amplify.github.io/docs/cli-toolchain/quickstart?sdk=js#concepts-1 and it's mostly straight forward but when he tries to do a `amplify pull` he gets 3 apps to select from (see second image) ... how should I proceed here?

Thanks!
![1](https://user-images.githubusercontent.com/638567/75168977-c0369400-56f5-11ea-979c-70e15a3435af.png)

![2](https://user-images.githubusercontent.com/638567/75170311-e826f700-56f7-11ea-9fa3-933fb0a17df5.png)

**Comment by nikhname** [comment]
@damianrr a couple questions
1. Did you create this project on the console or with the CLI?
2. Did you see the 3 apps after doing amplify pull and adding more environments?

With (2.) it would be very helpful for reproing if you could provide any other actions you did with the CLI before you saw the 3 apps on the console. Thanks!

**Comment by damianrr** [comment]
Hello @nikhname 
1. I created the project in the CLI
2. It's been a while since I'm seeing my app as 3 in the Console (way before the amplify pull for starters, also take into account that after I created the backend environments, like a year ago I continued seeing only one app in the Console, no problem, until a couple of month, maybe a little more where I suddenly noticed that now my Console reflected my App as 3 ). 
I do have upgraded my amplify cli package from time to time. Not sure if related to the issue or not.
